.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This silly file is the configuration for VSCode's debugger. Looks like it permits comments in its JSON markup. The debugger works, BTW! Was a big aid.